File: SPRINT MOBILE/OLD STUFF/INTEGRATION/HK SIKRAS 0.5 Integration/HuntKingdom/src/com/mycompany/entities/Espece.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.entities;

/**
 *
 * @author ilyess
 */
public class Espece {
     private  int idEspece;
    private String nomEspece;
    private String descriptionEspece;
    private String image;
    private String poids;
     private String type;
    private String zone;
    private String ville;
    private int idS;

    
    public Espece(int idEspece, String nomEspece, String descriptionEspece, String image, String poids, String type, String zone, String ville, int idS) {
        this.idEspece = idEspece;
        this.nomEspece = nomEspece;
        this.descriptionEspece = descriptionEspece;
        this.image = image;
        this.poids = poids;
        this.type = type;
        this.zone = zone;
        this.ville = ville;
        this.idS = idS;
    }

    public Espece(String nomEspece, String descriptionEspece, String image, String poids, String type, String zone, String ville, int idS) {
        this.nomEspece = nomEspece;
        this.descriptionEspece = descriptionEspece;
        this.image = image;
        this.poids = poids;
        this.type = type;
        this.zone = zone;
        this.ville = ville;
        this.idS = idS;
    }
    public Espece(String nomEspece, String descriptionEspece, String image, String poids, String type, String zone, String ville) {
        this.nomEspece = nomEspece;
        this.descriptionEspece = descriptionEspece;
        this.image = image;
        this.poids = poids;
        this.type = type;
        this.zone = zone;
        this.ville = ville;
       
    }
    public Espece() {
            }

    public int getIdEspece() {
        return idEspece;
    }

    public void setIdEspece(int idEspece) {
        this.idEspece = idEspece;
    }

    public String getNomEspece() {
        return nomEspece;
    }

    public void setNomEspece(String nomEspece) {
        this.nomEspece = nomEspece;
    }

    public String getDescriptionEspece() {
        return descriptionEspece;
    }

    public void setDescriptionEspece(String descriptionEspece) {
        this.descriptionEspece = descriptionEspece;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getPoids() {
        return poids;
    }

    public void setPoids(String poids) {
        this.poids = poids;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getZone() {
        return zone;
    }

    public void setZone(String zone) {
        this.zone = zone;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public int getIdS() {
        return idS;
    }

    public void setIdS(int idS) {
        this.idS = idS;
    }

    @Override
    public String toString() {
        return "Espece{" + "nomEspece=" + nomEspece + ", descriptionEspece=" + descriptionEspece + ", image=" + image + ", poids=" + poids + ", type=" + type + ", zone=" + zone + ", ville=" + ville + ", idS=" + idS + '}';
    }

    
    
    
}

File: HuntKingdom Web Symfony 3.4/src/EspeceBundle/Entity/espece.php
<?php

namespace EspeceBundle\Entity;
use Symfony\Component\HttpFoundation\File\File;
use Symfony\Component\Validator\Constraints as Assert;

use Doctrine\ORM\Mapping as ORM;

/**
 * espece
 *
 * @ORM\Table(name="espece")
 * @ORM\Entity(repositoryClass="EspeceBundle\Repository\especeRepository")
 */
class espece
{
    /**
     * @var int
     * @ORM\Column(name="idEspece", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $idEspece;

    /**
     * @var string
     *
     * @ORM\Column(name="nomEspece", type="string", length=255)
     */
    private $nomEspece;

    /**
     * @ORm\ManyToOne(targetEntity="saison")
     * @ORM\JoinColumn(name="idS",referencedColumnName="idSaison")
      */
     private $idS;

    /**
     * @var string
     *
     * @ORM\Column(name="descriptionEspece", type="string", length=255)
     */
    private $descriptionEspece;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255)
     * @Assert\Image()
     * @Assert\NotBlank(message="plz enter an image")
     */

    private $image;

    /**
     * @var string
     *
     * @ORM\Column(name="poids", type="string", length=255)
     */
    private $poids;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=255)
     */
    private $type;

    /**
     * @var string
     *
     * @ORM\Column(name="zone", type="string", length=255)
     */
    private $zone;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=255)
     */
    private $ville;

    /**
     * @return int
     */
    public function getIdEspece()
    {
        return $this->idEspece;
    }

    /**
     * @param int $idEspece
     */
    public function setIdEspece($idEspece)
    {
        $this->idEspece = $idEspece;
    }




    /**
     * Set nomEspece
     *
     * @param string $nomEspece
     *
     * @return espece
     */
    public function setNomEspece($nomEspece)
    {
        $this->nomEspece = $nomEspece;

        return $this;
    }

    /**
     * Get nomEspece
     *
     * @return string
     */
    public function getNomEspece()
    {
        return $this->nomEspece;
    }

    /**
     * Set descriptionEspece
     *
     * @param string $descriptionEspece
     *
     * @return espece
     */
    public function setDescriptionEspece($descriptionEspece)
    {
        $this->descriptionEspece = $descriptionEspece;

        return $this;
    }

    /**
     * Get descriptionEspece
     *
     * @return string
     */
    public function getDescriptionEspece()
    {
        return $this->descriptionEspece;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return espece
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * Set poids
     *
     * @param string $poids
     *
     * @return espece
     */
    public function setPoids($poids)
    {
        $this->poids = $poids;

        return $this;
    }

    /**
     * Get poids
     *
     * @return string
     */
    public function getPoids()
    {
        return $this->poids;
    }

    /**
     * Set type
     *
     * @param string $type
     *
     * @return espece
     */
    public function setType($type)
    {
        $this->type = $type;

        return $this;
    }

    /**
     * Get type
     *
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * Set zone
     *
     * @param string $zone
     *
     * @return espece
     */
    public function setZone($zone)
    {
        $this->zone = $zone;

        return $this;
    }

    /**
     * Get zone
     *
     * @return string
     */
    public function getZone()
    {
        return $this->zone;
    }

    /**
     * Set ville
     *
     * @param string $ville
     *
     * @return espece
     */
    public function setVille($ville)
    {
        $this->ville = $ville;

        return $this;
    }

    /**
     * Get ville
     *
     * @return string
     */
    public function getVille()
    {
        return $this->ville;
    }

    /**
     * @return mixed
     */
    public function getIdS()
    {
        return $this->idS;
    }

    /**
     * @param mixed $idS
     */
    public function setIdS($idS)
    {
        $this->idS = $idS;
    }



}
